Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -262,6 +262,11 @@ private final class TokenStreamCreator: SyntaxVisitor {
       after(node.identifier.lastToken, tokens: .space)
     }
 
+    let parameters = node.signature.input
+    let mustBreak = node.body != nil || node.signature.output != nil
+    after(parameters.leftParen, tokens: .break(.open, size: 0), .open(argumentListConsistency()))
```

Comment:
Can you factor these two lines out into a private helper function like `arrangeParameterClause` that takes the `ParameterClauseSyntax` and `forcesBreakBeforeRightParen` as arguments, and use that throughout? That'll reduce some duplication.

I had to do something similar when I moved around the function call argument list formatting: https://github.com/apple/swift-format/pull/88/files#diff-1935f1a8f63b8de81e00cb07ae1f6ebaR703

But your version can be simpler because we don't need to pass the delimiters in separately (that was necessary because for the call, it could be either parens or brackets).

Reply:
Done.

Reply:
Thanks!